src/notebooks/readme.md

```diff
@@ -0,0 +1,25 @@
+# notebooks
+
+These notebooks provide examples of how to work with the CSV files that are
+generated by the extractor programs.
+
+1. Ensure you have [Python 3.8+ and Poetry](../README.md#getting-started)

```

Comment:
```suggestion
1. Download the latest code from [the project homepage](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit) by clicking on the green "CODE" button and choosing an appropriate option. If choosing the Zip option, extract the file contents using your favorite zip tool.
1. Open a command prompt* and change to this file's directory (* e.g. cmd.exe, PowerShell, bash)
1. Ensure you have [Python 3.8+ and Poetry](../README.md#getting-started)
```

Reply:
@lalrg what do you think about the proposed text above?

Reply:
I think it is fine.